ui/lib/core/addon/components/toggle.js

```diff
@@ -15,29 +15,40 @@
  * @param {string} [status='normal'] - Status can be normal or success, which makes the switch have a blue background when checked=true
  */
 
-import Component from '@ember/component';
-import { computed } from '@ember/object';
+import Component from '@glimmer/component';
 import layout from '../templates/components/toggle';
+import { setComponentTemplate } from '@ember/component';
```

Comment:
Heads up that I don't think you need `setComponentTemplate` or to import the `layout` if you were to move the template file to the same directory as the component js file. The `autocomplete-input` component is an example of that in the core addon.

Reply:
Oh, let me try this!